lib/CMakeLists.txt

```diff
@@ -22,22 +21,33 @@ set ( PC_SOURCES
         pc_val.c
         )
 
+set ( LI_SOURCES
+        li_matrix.c
+        li_patch.c
+        li_point.c
+	)
```

Comment:
Initially, I did not like the idea of creating a separate variables for the `li_` files. But I now think this is a good idea. I'll do the same in the `Makefile`'s.